src/panels/config/automation/condition/types/ha-automation-condition-time.ts

```diff
@@ -1,24 +1,50 @@
 import "@polymer/paper-input/paper-input";
-import { customElement, html, LitElement, property } from "lit-element";
+import {
+  css,
+  CSSResult,
+  customElement,
+  html,
+  LitElement,
+  property,
+} from "lit-element";
+import { fireEvent } from "../../../../../common/dom/fire_event";
+import { computeRTLDirection } from "../../../../../common/util/compute_rtl";
+import "../../../../../components/ha-formfield";
+import { HaSwitch } from "../../../../../components/ha-switch";
 import { TimeCondition } from "../../../../../data/automation";
 import { HomeAssistant } from "../../../../../types";
 import {
   ConditionElement,
   handleChangeEvent,
 } from "../ha-automation-condition-row";
 
+interface Weekday {
+  name: string;
+  order: number;
+}
+
 @customElement("ha-automation-condition-time")
 export class HaTimeCondition extends LitElement implements ConditionElement {
   @property({ attribute: false }) public hass!: HomeAssistant;
 
-  @property() public condition!: TimeCondition;
+  @property({ attribute: false }) public condition!: TimeCondition;
+
+  private _days: Weekday[] = [
+    { name: "mon", order: 1 },
+    { name: "tue", order: 2 },
+    { name: "wed", order: 3 },
+    { name: "thu", order: 4 },
+    { name: "fri", order: 5 },
+    { name: "sat", order: 6 },
+    { name: "sun", order: 7 },
+  ];
```

Comment:
This doesn't change right?

This can be moved to a const

Reply:
@zsarnett to `src\common\const.ts`?

Reply:
Just to the top of the file for now

also we should make this an object I think

{
  mon: 1,
  tues: 2,
  ...
}

I think that would be faster and I don't think you need it to be an array right?

Reply:
ok, done. Please take a look now.